computeparticles/computeparticles.cpp

```diff
@@ -58,7 +59,7 @@ class VulkanExample : public VulkanExampleBase
 		float deltaT;
 		float destX;
 		float destY;
-		int32_t particleCount = PARTICLE_COUNT;
+		uint32_t particleCount = PARTICLE_COUNT;
```

Comment:
Yes, which is why I called it a "fix", but it's not related to the current change. Send it as a new PR if you want :]